RELEASING.md

```diff
@@ -5,5 +5,7 @@ To release a new version of Lightbeam:
 3. `git tag lightbeam-x.y.z`
 4. using the latest ESR release of the addon SDK, run `cfx xpi` to generate `lightbeam.xpi`
 5. log into <https://addons.mozilla.org> and click "Tools | Manage my submissions"
-6. click on "Lightbeam", then "Upload a new version" and select "full review" to upload the .xpi
-7. once the release is approved, push your changes to the repo: `git push; git push --tags`
+6. find "Lightbeam" and click "New version" to upload the .xpi
+7. add something like "Added feature foo. Full changelog on <a href="https://github.com/mozilla/lightbeam/issues?q=milestone%3A1.2.0+is%3Aclosed">Github</a>." in the version notes.
```

Comment:
In case someone forgets a milestone, diffing between release tags is probably more reliable.